src/testflinger_device_connectors/devices/oemrecovery/oemrecovery.py

```diff
@@ -21,6 +21,8 @@
 
 import yaml
 
+from wrapt_timeout_decorator import timeout
```

Comment:
I'm not really familiar with wrapt, but it sounds a little heavy-handed with the use of multiprocessing. I do like the approach of having a timeout decorator though. I wonder if we could just make our own and use it elsewhere in the device connectors? At the very least, I think we would need some testing of this with a realistic environment.

Reply:
I think we could make one, but maybe we can address this in another change that specifically for this purpose . Is that make sense to you?